packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -36,13 +36,15 @@
  rbac:
    enabled: true
    psp:
-@@ -85,3 +93,77 @@
+@@ -85,3 +93,79 @@
      additionalLabels: {}
      metricRelabelings: []
      relabelings: []
 +
 +disablePvc: true
 +
++systemdLogPath: "/var/log/journal"
```

Comment:
if we go this route we definitely need to document it, can you open/add-to an issue

Reply:
I want to test this a bit before comitting, assuming all goes well I will open an issue by eob

Reply:
This route is probably the best one anyway since the underlying distro has the choice of where this path can be. Documentation + this approach is what I would do